src/common/options.cc

```diff
@@ -2228,6 +2228,11 @@ std::vector<Option> get_global_options() {
 		     "only be rejected later, when they are used.")
     .set_flag(Option::FLAG_RUNTIME),
 
+    Option("mon_min_delay_socket", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
+    .set_default(false)
+    .set_description("priority packets between mons")
+    .add_service("mon"),
```

Comment:
I'd rename this option to `mon_use_min_delay_socket` and add a `see_also` for `osd_heartbeat_use_min_delay_socket`.

Reply:
done